src/main/java/com/emc/mongoose/storage/driver/pravega/PravegaStorageDriver.java

```diff
@@ -177,6 +178,23 @@ protected int submit(List<O> ops) throws InterruptRunException, IllegalStateExce
         return submit(ops, 0, ops.size());
     }
 
+	protected boolean invokePathDelete(final PathOperation<? extends PathItem> pathOp) {
+		final String path = pathOp.dstPath();
+		final String scopeName = path.substring(0,path.indexOf("/"));
+		final String streamName = path.substring(path.indexOf("/")+1, path.length());
+		if(streamManager.sealStream(scopeName, streamName)){
+			if(streamManager.deleteStream(scopeName, streamName)) {
+				return true;
+			}
+			else {
+				pathOp.status(Operation.Status.FAIL_UNKNOWN);
+				return false;
+			}
+		}
```

Comment:
"else": it would be useful to add the error message to the log like
```
Loggers.ERR.debug("Failed to seal the stream {} in the scope {}", stream, scope);
```